docs/publishing-a-release.md

```diff
@@ -0,0 +1,24 @@
+
+# Publishing a Release
+
+_These steps are only relevant to Sentry employees when preparing and publishing a new SDK release._
+
+**If you want to release a new SDK for the first time, be sure to follow the [New SDK Release Checklist](./new-sdk-release-checklist.md)**
+
+1. Determine what version will be released (we use [semver](https://semver.org)).
+2. Update [`CHANGELOG.md`](https://github.com/getsentry/sentry-javascript/edit/master/CHANGELOG.md) to add an entry for the next release number and a list of changes since the last release. (See details below.)
```

Comment:
the way I understand it, the changelog PR merge will trigger the merge from develop into master, right? Let's maybe add a comment to point this out. I think this might be valuable information for anyone coming into the repo.

Reply:
I updated it to point this out specifically, is that what you had in mind?

Reply:
Yes, I was thinking of exactly something like this. Thanks!